samples/bluetooth/classic/a2dp_sink/README.rst

```diff
@@ -0,0 +1,32 @@
+.. zephyr:code-sample:: bluetooth_a2dp_sink
+   :name: a2dp-sink
+   :relevant-api: bt_a2dp bluetooth
+
+   Use the A2DP APIs.
```

Comment:
You have the exact same description for the two samples - this won't look really good or helpful when folks will navigate the list of samples


```suggestion
   Use A2DP (Advanced Audio Distribution Profile) sink functionality.
```